src/webapp/components/current-data-submission/UploadsTable.tsx

```diff
@@ -28,7 +28,11 @@ export const UploadsTable: React.FC<UploadsTableProps> = ({ title, items, classN
                         <TableRow>
                             <TableCell>{i18n.t("Uploaded")}</TableCell>
                             <TableCell>{i18n.t("Period")}</TableCell>
-                            <TableCell>{i18n.t("Rows")}</TableCell>
+                            {currentModuleAccess.moduleName === "AMC" ? (
```

Comment:
instead of having checking the module directly, which would be adding biz logic to UI layer, can we add these details to moduleProperties. moduleProperties.primaryFileType does something similar. Same for code below.